frontend/src/components/ui/multi-select.tsx

```
@@ -0,0 +1,269 @@
+  };
+
+  return (
+    <TooltipProvider>
+      <div ref={containerRef} className="relative w-full" id={id}>
+        {/* Selected items display with count and tooltip */}
+        <Tooltip>
+          <TooltipTrigger asChild>
+            <div
+              className={`flex h-9 items-center justify-between px-3 border border-input rounded-md bg-background cursor-pointer text-sm ring-offset-background focus-within:ring-2 focus-within:ring-ring focus-within:ring-offset-2 ${className ?? ""}`}
+              onClick={handleToggleDropdown}
+            >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)